ZeroWasteFriendly/Scenes/Library/LibraryController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  LibraryController.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 27/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+  
+import UIKit
+
+protocol LibraryPresentable: SpinnerPresentable & AlertPresentable {
+    func displayPosts(posts: [PostDisplayable])
+}
+
+final class LibraryController: MainViewController {
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    private var postsDataSource = [PostDisplayable]()
+    
+    private let interactor: LibraryInteractor
+
+    init(interactor: LibraryInteractor) {
+        self.interactor = interactor
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupNavigationBar()
+        setupTableView()
+        interactor.getPosts()
+    }
+}
+
+private extension LibraryController {
+
+    private func setupTableView() {
+        tableView.register(cellType: LibraryTableViewCell.self)
+        tableView.separatorStyle = .none
```

Comment:
move this separator style to XIB
I'd do it in template too, as you mostly don't want separators